OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -71,6 +73,9 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Blugh I accidentally let this in whilst amending.

I'm not entirely sure what this is to be honest - VS adds it whenever I mess with the test project. I should research that at some point.